pkg/mark/markdown_test.go

```diff
@@ -19,24 +37,50 @@ func TestCompileMarkdown(t *testing.T) {
 	}
 
 	for _, filename := range testcases {
-		basename := filepath.Base(filename)
-		testname := strings.TrimSuffix(basename, ".md")
-		htmlname := filepath.Join(filepath.Dir(filename), testname+".html")
-
-		markdown, err := os.ReadFile(filename)
+		lib, err := stdlib.New(nil)
 		if err != nil {
 			panic(err)
 		}
-		html, err := os.ReadFile(htmlname)
+		markdown, htmlname, html := loadData(t, filename, "")
+		actual, _ := CompileMarkdown(markdown, lib, filename, "", 1.0, false, false)
+		test.EqualValues(string(html), actual, filename+" vs "+htmlname)
+	}
+}
+
+func TestCompileMarkdownDropH1(t *testing.T) {
```

Comment:
Also added tests for DropH1 - I can't swear to their accuracy, but they look right

Reply:
Thanks for adding these!